doc-content/shared-kie-docs/src/main/asciidoc/KieServer/securing-passwords-proc.adoc

```diff
@@ -0,0 +1,95 @@
+[id='securing-passwords-proc_{context}']
+
+= Securing {KIE_SERVER} passwords with a keystore
+
+{KIE_SERVER} uses passwords to authenticate users. For increased security, you can store {KIE_SERVER} passwords in a keystore. Because it supports symmetrics keys, you must use a Java Cryptography Extension KeyStore (JCEKS) with {KIE_SERVER}. Use KeyTool, which is part of the JDK installation, to create a new JCEKS.
+
+[NOTE]
+====
+If {KIE_SERVER} is not configured with JCEKS, {KIE_SERVER} stores passwords in configuration parameters in unsecured text files.
+====
+
+.Prerequisites
+
+* {KIE_SERVER} is installed in {EAP}.
+* Jave 8 or higher is installed.
+
+.Procedure
+. In the {EAP} home directory, enter the following command to create a {KIE_SERVER} user with the `kie-server` role and specify a password. In the following example, replace `<USER>` and `<PASSWORD>` with the user and password of your choice.
++
+ifdef::PAM,DM[]
+[source,bash]
+----
+$<EAP_HOME>/add-user.sh -a -e -u <USER> -p "<PASSWORD>" -g kie-server
+----
+endif::[]
+ifdef::DROOLS,JBPM[]
+[source,bash]
+----
+$<EAP_HOME>/add-user.sh -a -e -u <USER> -p "<PASSWORD>" -g kie-server
+----
+endif::[]
++
+. To use KeyTool to create a JCEKS, enter the following command in the Java 8 home directory:
++
+[source,bash]
+----
+${JAVA_HOME}/bin/keytool -importpassword -keystore <KEYSTORE_URL> -keypass <ALIAS_KEY_PASSWORD> -alias <PASSWORD_ALIAS> -storepass <KEYSTORE_PASSWORD> -storetype JCEKS
```

Comment:
Please change `KEYSTORE_URL` to `KEYSTORE_PATH`. The property should reference path where the keystore should be stored to.
Also please adjust description for the property below.